### ISSUE TITLE
Is there any way to check read coverage for each contig in p_ctg.fa

### ISSUE THREAD

**Issue by wjidea** [PacificBiosciences/FALCON#467]
Hi,

I would like to check the read coverage for each contig in the p_ctg assembly. I believe this information should be generated in the FALCON pipeline. I am wondering if there is a file to show the raw read coverage for each of the contig?

Thanks,

Jie

**Comment by rhallPB** [comment]
The raw read coverage isn't tracked through the falcon workflow, The most straightforward approach is to resequencing the raw reads against the assembly, a step required in polishing the final assembly anyway.

**Comment by wjidea** [comment]
Thanks for the reply. 

Do you mean I need to map the raw reads to the assembly using a mapper (e.g., blasr) ?

**Comment by rhallPB** [comment]
Yes, I would run the full resequencing workflow in order to generate a polished assembly. [https://github.com/PacificBiosciences/GenomicConsensus/blob/master/doc/HowTo.rst#running-a-large-scale-resequencingpolishing-job-in-smrtanalysis-23](https://github.com/PacificBiosciences/GenomicConsensus/blob/master/doc/HowTo.rst#running-a-large-scale-resequencingpolishing-job-in-smrtanalysis-23)
